src/Apps/Order/Routes/Payment/ExpressCheckoutPrototype/ExpressCheckoutProvider.tsx

```diff
@@ -6,6 +6,10 @@ import { getENV } from "Utils/getENV"
 export const ExpressCheckoutProvider = ({ order }) => {
   const { buyerTotalCents } = order
 
+  if (!buyerTotalCents) {
+    return null
+  }
```

Comment:
Looks like [this](https://github.com/artsy/force/blob/db4d317d99e9fb6166f9a57345632d67a8204bc1/src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx#L223-L232) might be the current way of displaying buyer total for both BN and MO order at the payment step.